bindings/python/benches/test_pt.py

```diff
@@ -152,3 +153,22 @@ def test_pt_sf_load_mps(benchmark):
         v = v.to(device="mps")
         tv = result[k]
         assert torch.allclose(v, tv)
+
+
+def test_pt_sf_save_cpu(benchmark):
+    weights = create_gpt2(12)
+
+    filename = "tmp.safetensors"
+
+    def setup():
+        try:
+            os.unlink(filename)
```

Comment:
I think it's worth adding a comment here, I'm sure we have forgotten why we do this in a year 😅.